### DISCUSSION TITLE
Disabling catppuccin

### DISCUSSION THREAD

**Discussion by rockorager** [LazyVim/LazyVim#1366] [Q&A]
Hey everyone, I am trying to disable 'catppuccin/nvim', and while I am able to disable many other plugins this one just won't.

This is my ./lua/plugins/plugins.lua file:

```lua
return {
  { "goolord/alpha-nvim", enabled = false },
  { "echasnovski/mini.ai", enabled = false },
  { "folke/persistence.nvim", enabled = false },
  { "nvim-pack/nvim-spectre", enabled = false },
  { "catppuccin/nvim", enabled = false },
  { "b0o/SchemaStore.nvim", enabled = false },
}
```

Everything else get's disabled, except `catppuccin/nvim`. Can confirm by checking the list of disabled plugins in `:Lazy`

Any ideas?

**Comment by dpetka2001** [comment]
That's because in the Lazyvim default [config](https://www.lazyvim.org/plugins/colorscheme#catppuccin) the `name` is changed. So you would just have to do `{ "catppuccin", enabled = false },`
